packages/f-services/src/validations.js

```diff
@@ -1,19 +1,31 @@
+/**
+ * Returns a deep object by traversing following the provided path.
+ *
+ * @param {Object} obj The object to traverse.
+ * @param {Array} path The path used to traverse the object.
+ * @return {Object} The object at the end of the path.
+ */
+const getDeepObjectByPath = (obj, path) => path.reduce((o, key) => (o && o[key] ? o[key] : null), obj);
```

Comment:
test for this method?

Reply:
Hadn't realised that this method wasn't exposed. For testing purposes and seeing as it's a key piece of inner functionality, i think exposing it might be a good idea here, that way if this method ever changes & breaks it will flag up in the tests. Happy to leave as is so it can be thought about later.